crates/pipeline-types/src/format/avro.rs

```diff
@@ -0,0 +1,47 @@
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+use utoipa::ToSchema;
+
+/// Avro output format configuration.
+#[derive(Serialize, Deserialize, Debug, Default, ToSchema)]
+#[serde(deny_unknown_fields)]
+pub struct AvroEncoderConfig {
+    /// Avro schema used to encode output records.
+    ///
+    /// Specified as a string containing schema definition in JSON format.
+    /// This schema must match precisely the SQL view definition, including
```

Comment:
which view?
doe the schema json contain the view name?

Reply:
Whatever view it is connected to. The schema does contain a name (it's just the name of the top-level record type in the schema), but the name doesn't matter for schema compatibility, it can differ from the name of the view.